### ISSUE TITLE
Connectivity table

### ISSUE THREAD

**Issue by ChayaSt** [openforcefield/cmiles#7]
Should cmiles also add the connectivity table as another identifier? The indices will be based on the map indices but 0 indexed rather than 1 indexed. Currently fragmenter generates a connectivity table for the molecule. 
pinging @dgasmith, @jchodera

**Comment by jchodera** [comment]
I'm not sure that's necessary. InChI already also includes a connectivity table encoded as one of its layers, I believe.

**Comment by ChayaSt** [comment]
@jchodera, as far as I understand, the connectivity in the standard InChI is tautomer-invariant valance bond connectivity so the hydrogens are not explicitly numbered there. The connectivity table will also include the hydrogens and single / double / triple / aromatic bond. 
Also, it is not clear what the standardization of the indices are.

**Comment by jchodera** [comment]
What's the utility of having the complete atomic connectivity table if we have this information already represented in other ways (such as the atom-tagged SMILES)?

**Comment by ChayaSt** [comment]
Yes, you can get the information from the atom-tagged SMILES so it's not essential. 

A better way to phrase this question is should we add this utility to cmiles? And if we do add something like this to cmiles utility, should we also have mapping from and to chemical graphs and SMILES here? 
Currently these utilities are in fragmenter. But it might make more sense to have them here.
